detekt.yml

```diff
@@ -84,7 +84,7 @@ formatting:
     active: true
     autoCorrect: true
   ExpressionBodySyntaxLineBreaks:
-      active: true
+      active: false
```

Comment:
ExpressionBodySyntaxLineBreaks is an auto-fix that's bugged. Instead of applying to every instance, it only applies to the first one per file. You have to run it multiple times to make it work.

Once those errors are gone, you'll get a lot of MaxLineLength ones. For that you just regenerate the baseline with `./gradlew detektEstablishAcceptedErrors` and push